build_cpp.sh

```diff
@@ -93,25 +93,33 @@ python2_inc=/usr/include/x86_64-linux-gnu/python2.7
 cd $prefix/catkin_ws
 
 echo
-echo -e '\e[34mRunning catkin_make.\e[39m'
+echo -e '\e[34mRunning catkin build.\e[39m'
 echo
 
 # Use ROS_PARALLEL_JOBS=-j1 to compile with only one core (Useful to point errors in catkin_make)
 export ROS_PARALLEL_JOBS="-j$PARALLEL_JOBS -l$PARALLEL_JOBS"
 # export ROS_PARALLEL_JOBS="-j1"
 
-catkin_make $VERBOSE --cmake-args -DUSE_CATKIN=ON -DCMAKE_TOOLCHAIN_FILE=$RBA_TOOLCHAIN -DANDROID_ABI=arm64-v8a -DANDROID_NATIVE_API_LEVEL=$platform $host64 \
-    -DPYTHON_EXECUTABLE=$python -DPYTHON_LIBRARY=$python_lib -DPYTHON_INCLUDE_DIR=$python_inc -DPYTHON_INCLUDE_DIR2=$python2_inc -DBUILD_SHARED_LIBS=0 -DCMAKE_INSTALL_PREFIX=$CMAKE_PREFIX_PATH -DBoost_NO_BOOST_CMAKE=ON -DBOOST_ROOT=$CMAKE_PREFIX_PATH -DANDROID=TRUE -DBOOST_INCLUDEDIR=$CMAKE_PREFIX_PATH/include/boost -DBOOST_LIBRARYDIR=$CMAKE_PREFIX_PATH/lib -DROSCONSOLE_BACKEND=print -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
+catkin config \
+  --extend $prefix/target \
+  --install-space $prefix/target \
+  --install \
+  --cmake-args \
```

Comment:
Please add `--merge-devel` or `--isolate-devel`.

catkin_tools has a very strange default-behavior (`--link-devel`) and symlinks files from a public devel-space to private devel-spaces, including the need to build a pseudo package initially to setup the devel-space, which also gets installed. The only purpose is to be able to clean individual packages from devel-space, which is not required here.

There are some issues with this method, e.g. if packages try to find artifacts from other related packages in `CATKIN_DEVEL_PREFIX`. This variable would point to the private devel-space then.